### PR TITLE
Add Basic Sku and Tag to IotHub

### PR DESCRIPTION
**Pull request by anusapan**
## Description

Add new cmdlet for updating tags of IotHub and add basic sku to it.

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/IotHub/Commands.IotHub/Commands.IotHub.csproj [hunk 1]
```diff
@@ -170,11 +164,19 @@
...
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="help\Update-AzureRmIotHub.md" />
     <None Include="MSSharedLibKey.snk" />
+    <None Include="packages.config" />
     <None Include="StartupScripts\*.ps1">
       <!-- <CopyToOutputDirectory>Always</CopyToOutputDirectory> -->
     </None>
   </ItemGroup>
+  <ItemGroup>
```
Comment (markcowl) [inline src/ResourceManager/IotHub/Commands.IotHub/Commands.IotHub.csproj]: You will need to make this change in the netcore csproj as well

Comment (anusapan) [reply src/ResourceManager/IotHub/Commands.IotHub/Commands.IotHub.csproj]: Done

**Comment by anusapan** [comment]
@markcowl Updated the PR

File: src/ResourceManager/IotHub/Commands.IotHub/help/New-AzureRmIotHubExportDevices.md [hunk 2]
```diff
@@ -50,6 +50,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ExcludeKeys
+{{Fill ExcludeKeys Description}}
```
Comment (markcowl) [inline src/ResourceManager/IotHub/Commands.IotHub/help/New-AzureRmIotHubExportDevices.md]: This needs a parameter description

File: src/ResourceManager/IotHub/Commands.IotHub/help/Update-AzureRmIotHub.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
+﻿---
+external help file: Microsoft.Azure.Commands.IotHub.dll-Help.xml
+Module Name: AzureRM.IotHub
+online version:
```
Comment (markcowl) [inline src/ResourceManager/IotHub/Commands.IotHub/help/Update-AzureRmIotHub.md]: online version should be: https://docs.microsoft.com/en-us/powershell/module/azurerm.iothub/update-azurermiothub

File: tools/HelpGeneration/Exceptions/ValidateHelpIssues.csv [hunk 4]
```diff
@@ -305,3 +305,4 @@ Update-AzureRmIoTDeviceProvisioningServiceAccessPolicy.md,No entry was found for
 Update-AzureRmIoTDeviceProvisioningServiceLinkedHub.md,No entry was found for the online version field in the header. The corresponding URL should be the following: https://docs.microsoft.com/en-us/powershell/module/azurerm.deviceprovisioningservices/update-azurermiotdeviceprovisioningservicelinkedhub
 Get-AzureRmIoTDeviceProvisioningServiceCertificateVerificationCode.md,No entry was found for the online version field in the header. The corresponding URL should be the following: https://docs.microsoft.com/en-us/powershell/module/azurerm.deviceprovisioningservices/get-azurermiotdeviceprovisioningservicecertificateverificationcode
 New-AzureRmIoTDeviceProvisioningServiceCertificateVerificationCode.md,No entry was found for the online version field in the header. The corresponding URL should be the following: https://docs.microsoft.com/en-us/powershell/module/azurerm.deviceprovisioningservices/new-azurermiotdeviceprovisioningservicecertificateverificationcode
+Update-AzureRmIotHub.md,No entry was found for the online version field in the header. The corresponding URL should be the following: https://docs.microsoft.com/en-us/powershell/module/azurerm.iothub/update-azurermiothub
```
Comment (markcowl) [inline tools/HelpGeneration/Exceptions/ValidateHelpIssues.csv]: Do not add this exception, instead, make the change as noted in the markdown file above

File: src/ResourceManager/IotHub/Commands.IotHub/help/New-AzureRmIotHubExportDevices.md (hunk 2 above)
Comment (anusapan) [reply src/ResourceManager/IotHub/Commands.IotHub/help/New-AzureRmIotHubExportDevices.md]: Done

File: src/ResourceManager/IotHub/Commands.IotHub/help/Update-AzureRmIotHub.md (hunk 3 above)
Comment (anusapan) [reply src/ResourceManager/IotHub/Commands.IotHub/help/Update-AzureRmIotHub.md]: Added

File: tools/HelpGeneration/Exceptions/ValidateHelpIssues.csv (hunk 4 above)
Comment (anusapan) [reply tools/HelpGeneration/Exceptions/ValidateHelpIssues.csv]: removed

**Comment by anusapan** [comment]
@markcowl Updated the PR and waiting for full test pass

**Comment by markcowl** [comment]
on demand run here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/775/

**Pull request merged**